src/toolchains.rs

```diff
@@ -231,7 +231,11 @@ impl Toolchain {
             .map_err(InstallError::Download)?;
         }
 
-        if dl_params.install_cargo {
+        // download cargo by default
+        // deactivate with the `--without-cargo` flag
+        // this default behavior was changed as of v0.6.0
```

Comment:
This line assumes a v0.6.0 release with the current changes if this PR is acceptable.